src/dns-test.js

```diff
@@ -3,7 +3,7 @@ const DNS_PACKET = require("dns-packet");
 const { v4: uuidv4 } = require("uuid");
 
 const APEX_DOMAIN_NAME = "dnssec-experiment-moz.net";
-const SMIMEA_DOMAIN_NAME = "8c6976e5b5410415bde908bd4dee15dfb167a9c873fc4bb8a81f6f2a._smimecert.dnssec-experiment-moz.net";
+const SMIMEA_DOMAIN_NAME = "9f86d081884c7d659a2feaa0c55ad015a3bf4f1b2b0b822cd15d6c15._smimecert.dnssec-experiment-moz.net";
```

Comment:
What was this change for?

Reply:
I forgot that a month or so ago we updated the domain name that corresponds to one of the DNS records we're requesting.